Source/CarthageKit/GitHub.swift

```diff
@@ -424,16 +424,17 @@ internal func loadGitHubAuthorization(forServer server: GitHubRepository.Server)
 
 /// Creates a request to fetch the given GitHub URL, optionally authenticating
 /// with the given credentials and content type.
-private func createGitHubRequest(URL: NSURL, _ authorizationHeaderValue: String?, contentType: String = APIContentType) -> NSURLRequest {
-	let request = NSMutableURLRequest(URL: URL)
-	request.setValue(contentType, forHTTPHeaderField: "Accept")
-	request.setValue(userAgent, forHTTPHeaderField: "User-Agent")
+public func createGitHubRequest(URL: NSURL, _ authorizationHeaderValue: String?, contentType: String = APIContentType) -> NSURLRequest {
```

Comment:
I agree. I think at the time, my reasoning was that I'd make it public, test it, then integrate that with the other tests for `releaseForTag`, and `downloadAsset` - which I did - but I never did the follow-up of "delete the tests for `createGitHubRequest` and make it private again".